packages/beacon-node/src/chain/blocks/verifyBlock.ts

```diff
@@ -174,6 +174,17 @@ export async function verifyBlocksInEpoch(
           default:
         }
       }
+
+      if (isForkPostFulu(fromForkBoundary.fork)) {
+        const fromBlobParameters = this.config.getBlobParameters(fromForkBoundary.epoch);
+        const toBlobParameters = this.config.getBlobParameters(toForkBoundary.epoch);
+
+        if (toBlobParameters.maxBlobsPerBlock !== fromBlobParameters.maxBlobsPerBlock) {
+          const {epoch, maxBlobsPerBlock} = toBlobParameters;
+
+          this.logger.info("Activating Blob Parameter Only (BPO) fork", {epoch, maxBlobsPerBlock});
```

Comment:
naybe just

```suggestion
          this.logger.info("Activating BPO fork", {epoch, maxBlobsPerBlock});
```

Reply:
> naybe just

Yea I think BPO is frequently used enough that users might know what it is referring to.